services/RabbitMQ.md

```diff
@@ -0,0 +1,27 @@
+# Guidelines for using RabbitMQ in MetaBrainz projects
```

Comment:
Since the title is general guidelines for RabbitMQ use, should we also add here to document things like acknowledgement mode ?

Reply:
Sure. I don’t know what it is but the guidelines are here for that.

Reply:
For the record, it has been added in commit https://github.com/metabrainz/guidelines/pull/17/commits/1df992397ea2dbef93917dd464cea84c60dca08e.